### PR TITLE
fix wsl 404s

### PR DESCRIPTION
**Pull request by thegagne**
Signed-off-by: David Gagne <undercoverdave@gmail.com>

**Description**

This PR fixes 404s in the WSL docs and fixes the k3d cluster create command.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/installation/wsl.md [hunk 1]
```diff
@@ -66,11 +66,11 @@ Once Docker is installed, the next step will be to install a Kubernetes cluster.
...
-export KUBECONFIG="$(k3d get-kubeconfig --name='k3s-default')"
+curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+k3d cluster create
+export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"
 ```
 
 ### 5. Finish up
 
-Follow the rest of Meshery [installation](/docs/pages/installation/index.md) steps.
+Follow the rest of Meshery [installation](/docs/installation) steps.
```
Comment (JubayerJoy) [inline docs/pages/installation/wsl.md:76]: Docs reference of WSL is now perfectly working!
Very nice work @thegagne! 🎈

File: docs/pages/installation/wsl.md [hunk 2]
```diff
@@ -66,11 +66,11 @@ Once Docker is installed, the next step will be to install a Kubernetes cluster.
 In this how-to, [K3d](https://github.com/rancher/k3d) will be used as it relys only on Docker.
 
 ```bash
-curl -s https://raw.githubusercontent.com/rancher/k3d/master/install.sh | bash
-k3d create
-export KUBECONFIG="$(k3d get-kubeconfig --name='k3s-default')"
+curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+k3d cluster create
+export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"
```
Comment (JubayerJoy) [inline docs/pages/installation/wsl.md:71]: @leecalcote, please review this part, looks good to me. @thegagne did a fabulous job!

**Review by JubayerJoy — COMMENTED** [review]
Very nice work @thegagne! 💯

**Comment by leecalcote** [comment]
Haha! Awesome, @thegagne. 🎈

**Comment by JubayerJoy** [comment]
Awesome work @thegagne!

**Pull request merged**